tor/core/validation.py

```diff
@@ -75,9 +104,17 @@ def verified_posted_transcript(post, config):
     for top_level_comment in linked_resource.comments.list():
         if (
             _author_check(post, top_level_comment) and
-            _header_check(top_level_comment, config)
+            _footer_check(top_level_comment, config)
         ):
             return True
 
     # Did their transcript get flagged by the spam filter? Check their history.
-    return _author_history_check(post, config)
+    if _author_history_check(post, config):
+        send_to_slack(
+            f'Found removed post: {post.submission.shortlink}',
+            config,
+            channel='#removed_posts'
```

Comment:
So nice! While slack might not be the authoritative, persistent database of choice, at least we can have some visibility into when posts are removed and not need to dig through the bot's logs. 👍